cmd/translatesfx/translatesfx/otel.go

```diff
@@ -160,16 +160,27 @@ func wrapStatement(stmt string, empty, negated bool) string {
 	return stmt
 }
 
-func globToRegexpStr(s string) (string, bool) {
+func saToRegexpStr(s string) (string, bool) {
 	negated := false
 	if strings.HasPrefix(s, "!") {
 		s = s[1:]
 		negated = true
 	}
+	if isRegexFilter(s) {
+		return s[1 : len(s)-1], negated
```

Comment:
Are we sure this can't be the empty string? `"/"` would be considered a regex filter as well?

Reply:
Like this?

```
metricsToExclude:
  - metricName: /
```

Reply:
Right, it's an edge case but wouldn't `isRegexFilter("/")` return true when it's not one?  The empty string would lead to an invalid index w/ the check as well.

Reply:
I see. Fixed, thanks.